docs/source/03_tutorial/07_set_up_experiment_tracking.md

```diff
@@ -0,0 +1,157 @@
+# Set up experiment tracking
+
+[Kedro-Viz](https://github.com/quantumblacklabs/kedro-viz) supports native experiment tracking from [version 4.1.0](https://github.com/quantumblacklabs/kedro-viz/releases/tag/v4.1.0) onwards. On having enabled Experiment tracking on your Kedro project, you will be able to access, edit and compare your runs directly from the Kedro-Viz web app.
+
+This tutorial will provide a step by step process to set up experiment tracking and access your logged metrics from each run on Kedro-Viz , using the starter project Kedro-spaceflights as outlined in [this tutorial](../03_tutorial/01_spaceflights_tutorial.md). You can also jump directly to [this section](../03_tutorial/02_experiment_tracking.md) for direct reference in setting up experiment tracking for your Kedro project.
+
+## Project setup
+
+We assume that you have already [installed Kedro](../02_get_started/02_install.md) and [Kedro-Viz](../03_tutorial/06_visualise_pipeline.md). Now set up a new Kedro project using the Kedro-spaceflight starter by running
+
+```bash
+kedro new --starter=spaceflights
+```
+
+When prompted for a project name, enter `Kedro Experiment Tracking Tutorial`. Subsequently, accept the default suggestions for `repo_name` and `python_package` by pressing enter.
+
+## Set up session store
+
+Within the domain of experiemnt tracking in Kedro, each Kedro run is considered a session. All logged metrics and details of that Kedro run (such as the timestamp, git username and branch) are stored within the session store database, which is a SQLite database that is automatically generated on your first Kedro run after setting up the session store in your Kedro project.
+
+To set up the session store, go to the `settings.py` file under `/src` and add the following:
+
+```python
+from kedro_viz.integrations.kedro.sqlite_store import SQLiteStore
+from pathlib import Path
+SESSION_STORE_CLASS = SQLiteStore
+SESSION_STORE_ARGS = {"path": str(Path(__file__).parents[2] / "data")}
+```
+
+## Set up tracking datasets
+
+There are 2 types of tracking datatsets: [`tracking.MetricsDataSet`](/kedro.extras.datasets.tracking.MetricsDataSet) and [`tracking.JSONDataSet`](/kedro.extras.datasets.tracking.JSONDataSet). The `tracking.MetricsDataSet` should be used for tracking numerical metrics, and the `tracking.JSONDataSet` can be used for tracking any other JSON-compatible data.
+
+Let's set up the following 2 datasets to log our r2 scores and parameters for each run by adding the following in `catalog.yml` under `/conf/base`:
```

Comment:
```suggestion
Let's set up the following two datasets to log `r2 scores` and `parameters` for each run by adding the following in `catalog.yml` under `/conf/base`:
```

Reply:
Should setting up catalog.yml come after the node and pipeline section?